web_viewer/index.html

```diff
@@ -267,8 +267,12 @@
           // Use the uncompressed size
           var content_length;
           var content_multiplier = 1;
-          // If the content is gzip encoded, try to get the uncompressed size.
-          if (response.headers.get("content-encoding") == "gzip") {
+          // If our custom size header is set, use that (see `re_web_viewer_server` crate)
```

Comment:
I came across this crate this morning btw: https://github.com/mathematic-inc/if-changed

Reply:
uhh that's pretty cool. But don't want to set this up in this pr now, creating a dev-experience issue

Reply:
https://github.com/rerun-io/rerun/issues/6062